index.js

```diff
@@ -54,8 +69,18 @@ function getHighestMetascore() {}
  *  getAverageIMDBRating(movies);
  *  //> 7.76
  */
-function getAverageIMDBRating() {}
-
+function getAverageIMDBRating(movies) {
```

Comment:
We should be doing some sort of check here to return 0 if there are no movies. Otherwise, we may error out or return something unexpected(NaN) on 82.